src/components/viewer2d/viewer2d.jsx

```diff
@@ -353,16 +353,16 @@ export default function Viewer2D(
         style={{ gridColumn: 2, gridRow: 2 }}
         width={width - rulerSize}
         height={height - rulerSize}
-        value={viewer2D.isEmpty() ? null : viewer2D.toJS()}
+        value={viewer2D.isEmpty() ? INITIAL_VALUE : viewer2D.toJS()}
         onChangeValue={onChangeValue}
         tool={mode2Tool(mode)}
         onChangeTool={onChangeTool}
         detectAutoPan={mode2DetectAutopan(mode)}
         onMouseDown={onMouseDown}
         onMouseMove={onMouseMove}
         onMouseUp={onMouseUp}
-        miniaturePosition="none"
-        toolbarPosition="none"
+        toolbarProps={{position:"none"}}
```

Comment:
Got this error with those value:

```
react.development.js:209 Warning: Failed prop type: Invalid prop `position` of value `none` supplied to `Ce`, expected one of ["top","right","bottom","left"].
    at Ce (webpack-internal:///../node_modules/react-svg-pan-zoom/build-umd/react-svg-pan-zoom.min.js:1:25279)
    at t (webpack-internal:///../node_modules/react-svg-pan-zoom/build-umd/react-svg-pan-zoom.min.js:1:31562)
    at div
    at Viewer2D (webpack-internal:///../src/components/viewer2d/viewer2d.jsx:106:20)
    at Content (webpack-internal:///../src/components/content.jsx:24:20)
    at div
    at ReactPlannerContent (webpack-internal:///../src/react-planner.jsx:51:21)
    at ReactPlanner (webpack-internal:///../src/react-planner.jsx:123:21)
    at ConnectFunction (webpack-internal:///../node_modules/react-redux/es/components/connectAdvanced.js:234:68)
    at div
    at Provider (webpack-internal:///../node_modules/react-redux/es/components/Provider.js:19:20)
    at Wrapper (webpack-internal:///./src/wrapper.jsx:73:64)
    ```

Reply:
<img width="1162" alt="image" src="https://github.com/Luccid-ai/react-planner/assets/32320149/509b01a5-4872-4f81-9f07-a0627ab8602a">

Did you upgrade ?
```
"react-svg-pan-zoom": "^3.12.1",
```

Reply:
sorry, forgot to update indeed... Thanks again for the contribution!